dijet/selection/trigger.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+
+from typing import Tuple
+from columnflow.util import maybe_import
+from columnflow.selection import Selector, SelectionResult, selector
+from columnflow.production import Producer
+
+ak = maybe_import("awkward")
+np = maybe_import("numpy")
+
+
+def masked_sorted_indices(mask: ak.Array, sort_var: ak.Array, ascending: bool = False) -> ak.Array:
+    """
+    Helper function to obtain the correct indices of an object mask
+    """
+    indices = ak.argsort(sort_var, axis=-1, ascending=ascending)
+    return indices[mask[indices]]
+
+
+# TODO: Use self in uses. Follow:
+#       https://github.com/uhh-cms/topsf/blob/11126442981a25663c60767e56c5406afc9d4f7b/topsf/production/probe_jet.py#L163-L192
+@selector(
+    uses={
+        "dijets.pt_avg", "probe_jet.eta",
+    },
+    exposed=True,
+)
+def trigger_selection(
+    self: Selector,
+    events: ak.Array,
+    **kwargs,
+) -> Tuple[ak.Array, SelectionResult]:
+
+    # TODO: Work at:
+    #       - central vs. forward
+    #       - different years
+    #       - SingleJet
+    #       - Jet collections (AK4 vs. AK8)
+
+    # per-event trigger index (based on thresholds)
+    thrs = self.config_inst.x.trigger_thresholds.dijet.central.UL17
+    sel_trigger_index = np.digitize(ak.to_numpy(events.dijets.pt_avg), thrs) - 1  # can be -1
+
+    # mask -1 values to avoid picking wrong trigger. Does not cut events but marks them as unvalid!
+    sel_trigger_index = ak.mask(sel_trigger_index, sel_trigger_index < 0, valid_when=False)
+
+    # put trigger decisions into 2D array
+    pass_triggers = []
+    for trigger_name in self.config_inst.x.triggers.dijet.central:
```

Comment:
Should add a check that the number of thresholds configured corresponds to the number of triggers
```suggestion
    # sanity check: number of thresholds matches the triggers
    trigger_names = self.config_inst.x.triggers.dijet.central
    assert len(trigger_names) == len(thrs)

    # put trigger decisions into 2D array
    pass_triggers = []
    for trigger_name in trigger_names:
```